daemon/container_windows.go

```diff
@@ -193,6 +185,7 @@ func (container *Container) PrepareStorage() error {
 	return nil
 }
 
+// CleanupStorage unprepares the layer after shutdown? FIXME
```

Comment:
prepareStorage is an extra step required on the Windows platform after mounting to ensure the storage is ready for use.

cleanupStorage is an extra step required on the Windows platform to ensure the storage is cleaned up before the storage is ready for use.

Note that these are being removed shortly to tidy up the Windows graph driver.